frontends/PyCDE/src/pycde/module.py

```diff
@@ -183,9 +194,12 @@ def __init__(self, *args, **kwargs):
           value = support.get_value(inputs[name])
           assert value is not None
         else:
-          backedge = BackedgeBuilder.current().create(type, name, self)
-          self.backedges[idx] = backedge
-          value = backedge.result
+          if extern:
+            value = zeroconst(type)
+          else:
```

Comment:
This is my only concern. Why not make this conditioned on `if not extern`, and drop the part that sets `value = zeroconst(type)`? Does the verifier reject it or something?

Reply:
Nothing complains, but a line below appends it to a list so I'm pretty sure we'll have an off-by-one port error.